tests/scripts/all.sh

```diff
@@ -1030,6 +1030,21 @@ component_test_no_64bit_multiplication () {
     make test
 }
 
+component_build_uecc_baremetal_cmake () {
+    msg "build: uecc baremetal native, cmake"
+    scripts/config.pl baremetal
+    scripts/config.pl set MBEDTLS_USE_UECC
+    CC=gcc cmake .
+    make
+}
+
+component_build_uecc_baremetal_make () {
+    msg "build: uecc baremetal native, make"
+    scripts/config.pl baremetal
+    scripts/config.pl set MBEDTLS_USE_UECC
+    make CC=gcc CFLAGS='-Werror -Wall -Wextra'
```

Comment:
Those flags are consistent with other builds in `all.sh`, but actually the other uses are mistaken (there was a PR to fix this long ago, but apparently it got lost). The proper way is `CFLAGS='-Werror -O1'`. Rationale: `-Wall -Wextra` are redundant as they're already set in `WARNING_CFLAGS` by default, but `-O1` is useful because some warnings are not found at `-O0` (and nromally `CFLAGS` defaults to `-O2` but we're overwriting that variable, so we need to put re-enable some optimisation as well.)

Reply:
Done